proposals/stringref/Overview.md

```diff
@@ -479,6 +479,55 @@ Return the number of codepoints that were actually consumed.
 Return a substring of *`view`*, starting at the current position of
 *`view`* and continuing for at most *`codepoints`* codepoints.
 
+## Binary encoding
+
+```
+reftype ::= ...
+         |  0x65 ⇒ stringref         ; SLEB128(-0x1b)
+         |  0x64 ⇒ stringview_wtf8   ; SLEB128(-0x1c)
+         |  0x63 ⇒ stringview_wtf16  ; SLEB128(-0x1d)
+         |  0x62 ⇒ stringview_iter   ; SLEB128(-0x1e)
+
+wtf8_policy ::= 0x00 ⇒ utf8
+             |  0x01 ⇒ wtf8
+             |  0x02 ⇒ replace
+
+instr ::= ...
+       |  0xfb 0x80 $mem:u32              ⇒ string.new_wtf8 $mem
+       |  0xfb 0x81 $mem:u32              ⇒ string.new_wtf16 $mem
```

Comment:
I believe it's:
```
0xfb 0x80 $mem:u32 $len:u32              ⇒ string.new_wtf8 $mem $len
0xfb 0x81 $mem:u32 $len:u32              ⇒ string.new_wtf16 $mem $len
```
possibly in other instruction $len might be missing, unsure.

Reply:
Sorry, sloppy notation -- the `$` prefix is meant to indicate an immediate.  In the case it's which memory to use.  The address and length are taken from the operand stack and aren't part of the encoding itself.  Hope that is clear.